zeroconf/__init__.py

```diff
@@ -2290,6 +2290,13 @@ def add_multicast_member(
         elif _errno in err_einval:
             log.info('Interface of %s does not support multicast, ' 'it is expected in WSL', interface)
             return None
+        elif _errno == 10042:
```

Comment:
Is there a constant in the `errno` module that can be used instead of this value?

Reply:
Yes I hadn't spotted it, WSAENOPROTOOPT